pytest_structlog.py

```diff
@@ -61,6 +73,30 @@ def has(self, message, **context):
         context["event"] = message
         return any(is_submap(context, e) for e in self.events)
 
+    def log(self, level, event, **kw):
+        """Create log event to assert against"""
+        return {"level": level_to_name(level), "event": event, **kw}
```

Comment:
For older Python versions compat 

```suggestion
        result = {"level": level_to_name(level), "event": event}
        result.update(kw)
        return result
```